src/cmds/qsub_sup.c

```diff
@@ -1424,7 +1424,7 @@ get_comm_filename(char *fname)
 
 	count = snprintf(fname, MAXPIPENAME, "%s/pbs_%.16s_%lu_%.8s_%.32s_%.16s_%.5s",
 			 tmpdir,
-			 ((server_out == NULL || server_out[0] == 0) ? "default" : server_out),
+			 server_out[0] == 0 ? "default" : server_out,
```

Comment:
Using the null char literal '\0' makes the code a bit more readable, since it makes it clear that we're dealing with a string and not an array of integers.

Reply:
changed